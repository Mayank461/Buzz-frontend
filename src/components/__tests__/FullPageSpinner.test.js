import renderer from 'react-test-renderer';
import FullPageSpinner from '../FullPageSpinner';
test('should render FullPageSpinner Panel', () => {
  const tree = renderer.create(<FullPageSpinner />).toJSON();
  expect(tree).toMatchSnapshot();
});
