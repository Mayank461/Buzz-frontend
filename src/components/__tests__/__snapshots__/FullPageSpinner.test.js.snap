// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FullPageSpinner Panel 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "background": "#fff",
      "display": "flex",
      "height": "100vh",
      "left": 0,
      "position": "fixed",
      "right": 0,
      "top": 0,
      "zIndex": 1000,
    }
  }
>
  <div
    className="spinner-border text-dark position-absolute start-50"
  >
    <span
      className="visually-hidden"
    >
      Loading...
    </span>
  </div>
</div>
`;
